src/App/useGetConfig.ts

```diff
@@ -4,11 +4,13 @@ interface Config {
   publicRead: boolean
 }
 
-const useGetConfig = () => {
-  const [config, setConfig] = useState<Config | null>(null)
+const useGetConfig = (): Config => {
+  const [config, setConfig] = useState<Config>({ publicRead: false })
```

Comment:
I believe that a default value of null or undefined is better here, instead of trying to make a safe assumption before the server has provided the real answer. This is especially true if there are more configs showing up in the future.

Reply:
That's fair.

tbh, I was just being lazy and didn't want to handle null in the component consuming this. 😏

Reply:
Maybe an empty object would be a good compromise since this project has optional chaining enabled?

Reply:
I would still go for undefined as default value, and make the members non-optional. Then do a 

```tsx
{!config ? <p>Loading...</span> : <div>....</div>}
```

I would prefer if there was no UI until the configuration is available. What kind of loading indication is used can be improved to some kind of spinner later.